test/run-tests

```diff
@@ -550,7 +559,17 @@ class Task:
 
             _playbook_set = [{"is_collection": False, "playbooks": playbooks}]
 
+            # If test_collections is true, converts the role to the collections format.
             if test_collections:
+                from importlib import import_module
+
+                _lsr = import_module("lsr_role2collection")
+                _lsr.role2collection()
+                collection_paths = (
+                    os.environ["COLLECTION_DEST_PATH"]
+                    + ":~/.ansible/collections:/usr/share/ansible/collections"
+                )
```

Comment:
You don't have to do this for every iteration - you only have to do this once

Reply:
Once per pull request == task.run, isn't it?